src/components/utils/src/signals_win.cc

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
@pestOO, what do you mean? We have similar signals_posix file for posix signals handling implementation.